src/main/java/edu/unc/lib/boxc/migration/cdm/util/DisplayProgressUtil.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.migration.cdm.util;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.repeat;
+
+/**
+ * Utilities for displaying progress in a CLI
+ *
+ * @author bbpennel
+ */
+public class DisplayProgressUtil {
+    private static final int PROGRESS_BAR_UNITS = 40;
+    private static final double PROGRESS_BAR_DIVIDOR = (double) 100 / PROGRESS_BAR_UNITS;
+
+    /**
+     * Render a progress bar, percent, and total
+     *
+     * @param current
+     * @param total
+     */
+    public static void displayProgress(long current, long total) {
```

Comment:
maybe it would be good to include a comment that shows what the progress bar looks like? it's hard to tell from this code